src/osd/osd_types.cc

```diff
@@ -100,6 +100,12 @@ const char * ceph_osd_op_flag_name(unsigned flag)
     case CEPH_OSD_OP_FLAG_FADVISE_NOCACHE:
       name = "fadvise_nocache";
       break;
+    case CEPH_OSD_OP_FLAG_WITH_REFERENCE:
+      name = "with_reference";
+      break;
```

Comment:
These 3 lines should be in a separate commit, in case it gets backported and this pull request doesn't.

Reply:
It's reasonable,  thanks! New version will be sent soon.